assets/scripts/main.js

```diff
@@ -30,6 +30,10 @@ import './dialogs/_save_as_image'
 
 // Street
 import './streets/name'
+import './streets/scroll'
+
+// Gallery
+import './gallery/scroll'
```

Comment:
Yeah, this is a little sloppy. Scripts are executed only one time when they are `import`ed, so subsequent `import`s are references to the original import, and not re-run. (This is not the case if they are separate bundles, as in the case of `main.js` and `preinit.js`.) So when I put something in `exports.js` I did not also add it in `main.js`. So far, when a script no longer needs to be exported to the `window`, I just move that `import` line into `main.js`.

I'd be happy to reconsider this process to make it easier to maintain.

`./util/fetch_blocking.js` is only imported in `exports.js`. Looks like the scripts that use its functionality have not been ported yet.